ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -1447,7 +1447,10 @@ extension BrowserViewController: WKUIDelegate {
     let actionProvider: UIContextMenuActionProvider = { _ -> UIMenu? in
       var actions = [UIAction]()
 
-      if let currentTab = self.tabManager.selectedTab {
+      if let currentTab = self.tabManager.selectedTab,
+        let contextHelper = currentTab.getContentScript(name: ContextMenuHandler.scriptName)
+          as? ContextMenuHandler
+      {
```

Comment:
Maybe lets move this contextHelper if-let to where it's used when making the `saveImageAction`